AstronomyObjC/AstronomyObjC/Models/TMCMarsPhotoReference.h

```diff
@@ -0,0 +1,28 @@
+//
+//  TMCMarsPhotoReference.h
+//  AstronomyObjC
+//
+//  Created by Christopher Devito on 5/18/20.
+//  Copyright © 2020 Christopher Devito. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "TMCMarsRoverCamera.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface TMCMarsPhotoReference : NSObject
+@property NSNumber *identification;
```

Comment:
I generally like to put an extra newline between these two just to help tell where things begin & end.